fairlearn/metrics/_dashboard_dictionary.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from ._group_metric_set import create_group_metric_set
+
+
+def create_dashboard_dictionary(
+        model_type,
+        y_true,
+        models,
```

Comment:
Will be changing to `prediction_type` and `predictions` - @MiroDudik pointed out that 'model' tends to have a rather specific meaning in the ML community, and this usage didn't match that.